Description: Jammy shellcheck fixes
 Another new release, another set of shellcheck issues detected needing
 fixing.

Author: Benjamin M Romer <benjamin.romer@canonical.com>
Last-Update: 2022-05-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/kpatch-build/kpatch-build
+++ b/kpatch-build/kpatch-build
@@ -446,7 +446,7 @@
 	dir="$(dirname "$1")"
 	absdir="$(readlink -f "$dir")"
 	pwddir="$(readlink -f .)"
-	pdir="${absdir#$pwddir/}"
+	pdir="${absdir#"$pwddir"/}"
 	file="$(basename "$1")"
 	grepname="${1%.o}"
 	grepname="$grepname\\.o"
--- a/kpatch-build/kpatch-cc
+++ b/kpatch-build/kpatch-cc
@@ -23,7 +23,7 @@
 			[[ "$obj" = */.tmp_mc_*.o ]] && break;
 
 			[[ "$obj" = */.tmp_*.o ]] && obj="${obj/.tmp_/}"
-			relobj=${obj##$KPATCH_GCC_SRCDIR/}
+			relobj=${obj##"$KPATCH_GCC_SRCDIR"/}
 			case "$relobj" in
 				*.mod.o|\
 				*built-in.o|\
